src/status.rs

```diff
@@ -1,9 +1,12 @@
 use crate::timeformat::MinSec;
 use crate::worker::StatusUpdater;
+use chrono::Local;
```

Comment:
Importing a name like `Local` can be a bit unclear in the rest of the code. Maybe just use `chrono::Local` when you use it, or give it a more specific name here like `use chrono::Local as LocalTime` or something.